soh/soh/Enhancements/randomizer/randomizer_check_tracker.cpp

```diff
@@ -110,7 +110,8 @@ std::vector<uint32_t> buttons = { BTN_A, BTN_B, BTN_CUP,   BTN_CDOWN, BTN_CLEFT,
 
 void DrawCheckTracker(bool& open) {
     if (!open) {
-        CVarSetInteger("gCheckTrackerEnabled", 0);
+        CVarClear("gCheckTrackerEnabled");
```

Comment:
Looks like you missed a check for the CVar being true before clearing/saving. Otherwise this is requesting a save on tick for every frame when the check tracker is closed.

Reply:
Good catch!